src/lib/ui/containers/SignupForm/SignupForm.svelte

```diff
@@ -9,14 +9,14 @@
   } from "@tuentyfaiv/svelte-form";
   import { getTexts } from "$lib/logic/utils/objects.js";
   import { fieldsSignup } from "$lib/logic/schemas/index.js";
+  import { useGlobalFormStyles } from "$lib/logic/stores/config.js";
 
-  import type { Readable } from "svelte/store";
   import type { FieldInputForm } from "$lib/logic/typing/globals/interfaces.js";
   import type { SignupValues } from "$lib/logic/typing/schemas/auth.js";
-  import type { ConfigForm } from "$lib/logic/typing/store/config.js";
   import type { Props, SignupFields } from "./SignupForm.proptypes.js";
 
   import * as stylesinternal from "./SignupForm.styles.js";
+  import { cx } from "@emotion/css";
```

Comment:
🚫 **[eslint]** <[import/order](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/order.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`@emotion/css` import should occur before import of `$lib/logic/utils/objects.js`
```suggestion
  import { cx } from "@emotion/css";
  import { getTexts } from "$lib/logic/utils/objects.js";
  import { fieldsSignup } from "$lib/logic/schemas/index.js";
  import { useGlobalFormStyles } from "$lib/logic/stores/config.js";

  import type { FieldInputForm } from "$lib/logic/typing/globals/interfaces.js";
  import type { SignupValues } from "$lib/logic/typing/schemas/auth.js";
  import type { Props, SignupFields } from "./SignupForm.proptypes.js";

  import * as stylesinternal from "./SignupForm.styles.js";
```